opencivicdata/migrations/0002_auto_20150131_1021.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('opencivicdata', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='person',
+            name='family_name',
+            field=models.CharField(max_length=100, default=''),
```

Comment:
We should likely allow for NULL - I'd love to keep this NULL for data we don't have (e.g. not empty-string, which could perhaps be a valid value here)